src/main/java/frc/robot/Auto.java

```diff
@@ -16,11 +16,13 @@ public class Auto {
     private Limelight limelight;
     private static Gyro gyro = new Gyro();
     private Dashboard dashboard;
+    private Arm arm;
 
     public Auto(Drivetrain drivetrain, Limelight limelight, Dashboard dashboard) {
         this.drivetrain = drivetrain;
         this.limelight = limelight;
         this.dashboard = dashboard;
+        this.arm = arm;
```

Comment:
This doesn't do anything.  Probably you meant to add a new parameter named "arm" to this constructor.